examples/guru2d1_adjoint.cpp

```diff
@@ -0,0 +1,80 @@
+#include <finufft.h>
+
+#include <complex>
+#include <iomanip>
+#include <iostream>
+#include <vector>
+using namespace std;
+
+int main(int argc, char *argv[]) {
+  /* 2D type 1 guru interface example of calling the FINUFFT library from C++,
+     using STL double complex vectors, with a math test. Similar to simple2d1
+     except illustrates the guru interface.
+     To compile, see README.  Usage: ./guru2d1
```

Comment:
update this so not simply a copy of guru2d1 comments.  explain it does the adj t2 with sign flip.

Reply:
I just fixed in 01e88ffc